src/native-state-drm.cpp

```diff
@@ -236,8 +255,9 @@ udev_get_node_that_pass_in_enum(
             if (check_passed) {
                 const char * device_node_path =
                     udev_device_get_devnode(current_device);
+                bool is_modeset = udev_drm_test_modeset(device_node_path);
```

Comment:
Since std::strings can't be created from null char* strings, we need to guard against this happening, i.e., only call udev_drm_test_modeset if the path is not null.